src/Umbraco.Tests/FrontEnd/UmbracoHelperTests.cs

```diff
@@ -118,21 +131,19 @@ public void Truncate_By_Words_With_Tag()
         }
 
         [Test]
-        public void Truncate_By_Words_Mid_Tag()
+        public static void Truncate_By_Words_Mid_Tag()
         {
-            var text = "Hello world, this is some text <a href='blah'>with a link</a>";
-
             var helper = new UmbracoHelper();
 
-            var result = helper.TruncateByWords(text, 7).ToString();
+            var result = helper.TruncateByWords(SAMPLE, 7).ToString();
 
             Assert.AreEqual("Hello world, this is some text <a href='blah'>with&hellip;</a>", result);
         }
 
         [Test]
-        public void Strip_All_Html()
+        public static void Strip_All_Html()
         {
-            var text = "Hello world, <b>this</b> is some text <a href='blah'>with a link</a>";
+            const string text = "Hello world, <b>this</b> is some text <a href='blah'>with a link</a>";
```

Comment:
This string with the `<b>` tag in it repeats a few times, should it also be a field, like `Sample`?